programs/drift/src/controller/orders.rs

```diff
@@ -1027,129 +1025,113 @@ fn sanitize_maker_order<'a>(
         Some(maker_stats)
     };
 
+    if maker.is_being_liquidated() || maker.is_bankrupt() {
+        return Ok((None, None, None, None));
+    }
+
     let market = perp_market_map.get_ref(&taker_order.market_index)?;
 
-    let maker_order_id = maker_order_id.ok_or(ErrorCode::MakerOrderNotFound)?;
-    let maker_order_index = match maker.get_order_index(maker_order_id) {
-        Ok(order_index) => order_index,
-        Err(_) => {
-            msg!("Maker has no order id {}", maker_order_id);
-            let fallback_order_index = find_fallback_maker_order(
-                &maker,
-                match taker_order.direction {
-                    PositionDirection::Long => &PositionDirection::Short,
-                    PositionDirection::Short => &PositionDirection::Long,
-                },
-                &MarketType::Perp,
-                taker_order.market_index,
-                Some(oracle_price),
-                slot,
-                market.amm.order_tick_size,
-            )?;
+    let mut maker_order_indexes = find_maker_orders(
```

Comment:
`maker_order_indexes` do we wanna rename this since its the price and index?

Reply:
yeah can do